content/en/docs/reference/node/kubelet-checkpoint-api.md

```diff
@@ -27,33 +27,63 @@ system all memory pages will be readable by the owner of the checkpoint archive.
 
 ## Security Risks and Mitigation Strategies
 
-1. **Exposure of sensitive data**: When a container is checkpointed, all memory pages, including private data and encryption keys, are saved to the local disk. If the checkpoint archive is accessed by unauthorized users, it can lead to data exposure and potential security breaches. Mitigation strategies include:
-
-   - Restricting access: Ensure that the checkpoint archive is accessible only by authorized users. Set appropriate file permissions and access controls to limit access to the archive.
+1. **Exposure of sensitive data**: When a container is checkpointed, all memory pages,
+ including private data and encryption keys, are saved to the local disk. If the 
+ checkpoint archive is accessed by unauthorized users, it can lead to data exposure 
+ and potential security breaches. Mitigation strategies include:
```

Comment:
```suggestion
1. **Exposure of sensitive data**: When a container is checkpointed, all memory pages,
   including private data and encryption keys, are saved to the local disk. If the 
   checkpoint archive is accessed by unauthorized users, it can lead to data exposure 
   and potential security breaches. The mitigation strategies include:
```

Reply:
Please indent the lines properly so the markdown paragraphs are clearer for future revisions.